notebooks/community/managed_notebooks/Recommender Systems/Recommender-system_notebook.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    PROJECT = "your-project-name"
</pre><p>Review Cloud AI Testing for variable substitution. This is incorrect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>